hr_timesheet_role/models/account_analytic_line.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    role_id = fields.Many2one(
+        comodel_name='project.role',
+        string='Role',
+        required=False,
```

Comment:
@pedrobaeza ^ it's not required at DB level right now

Reply:
Then it shouldn't be a problem to be installed with `hr_timesheet_sheet`...

Reply:
@bjeficent can you please elaborate on issue you're having?